scene/gui/graph_edit.cpp

```diff
@@ -2374,7 +2761,7 @@ GraphEdit::GraphEdit() {
 	top_layer->connect("focus_exited", callable_mp(panner.ptr(), &ViewPanner::release_pan_key));
 
 	connections_layer = memnew(Control);
-	add_child(connections_layer, false, INTERNAL_MODE_FRONT);
+	add_child(connections_layer, false);
```

Comment:
Why did you remove internal mode? It makes the connection layer easily accessible by the user.

Reply:
I tried several approaches (e.g. using the z-index plus some workarounds) and this was actually the only viable solution. I don't see how can possibly fix this without a major comfpat breakage later or introducing internal nodes at arbitrary positions.